server/src/main/java/fr/uga/l3miage/tp2/exo1/models/DeveloperMiageEntity.java

```diff
@@ -0,0 +1,19 @@
+package fr.uga.l3miage.tp2.exo1.models;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+@Table(name = "developer_miage")
+public class DeveloperMiageEntity {
+    @Id
+    @Column(name = "lastname")
+    private String lastname ;
+    @Column(name = "firstname")
+    private String firstname ;
+    @Column(name = "email")
+    private String email ;
+    @ManyToMany(mappedBy = "developers")
+    private Set <ProjetEntity> projets ;
```

Comment:
votre relation est à l'envers, le `mappedBy` devrait être dans l'entité `ProjectEntity` car la actuellement vous avez une table d'association qui ce nomme `projects_developper_miage` alors que je voulais une table qui commence `developper_miage_projects`
```suggestion
    @ManyToMany
    private Set <ProjetEntity> projets ;
```